package.json

```diff
@@ -136,6 +136,7 @@
   "resolutions": {
     "glob-parent": "^5.1.2",
     "codeceptjs/**/ansi-regex": "^4.1.1",
-    "codeceptjs/**/minimatch": "^3.0.5"
+    "codeceptjs/**/minimatch": "^3.0.5",
+    "flat": "^5.0.1"
```

Comment:
This looks like it will work, but did you try bumping the whole mocha dependency?  8.x is rather old and maybe that would make us more future proof?

Reply:
Hmm, yeah. I am going to try that